content/post/introducing-custom-container/index.md

```diff
@@ -0,0 +1,165 @@
+---
+title: A workaround of adding custom container to KubeDB managed Databases
+date: 2022-06-24
+weight: 25
+authors:
+  - Raihan Khan
+tags:
+  - kubeDB
+  - container
+  - MySQL
+  - Filebeat
+  - StatefulSet
+  - MySQLOpsRequest
+---
+
+### Add custom containers in KubeDB managed Database
+
+Let's assume you have a KubeDB managed Database deployed in your Kubernetes environment. Now, You want to inject a sidecar container in the database StatefulSet in order to extend and enhance the functionality of existing containers. Currently, KubeDB doesn't have support for custom container insertion. But, we are having a workaround to run KubeDB managed pods with your own custom container. In this demo, we are going to demonstrate how to inject Filebeat container in KubeDB managed MySQL cluster.  
```

Comment:
But, we are having a workaround to run KubeDB-managed pods with your own custom container.
---->

But, we will discuss a workaround to run a custom container along with the managed containers.

Reply:
Add a goal:

Suppose, you are running a KubeDB-managed MySQL database, and you need to monitor the general query logs. When you enable the mySQL query logs, it will write those logs in the ..., will need a tool like filebeat to export those logs to some remote monitoring solution (such as Elasticsearch). 

To achieve this, we will need to run filebeat container as a sidecar container....

Reply:
done